sodetlib/stream.py

```diff
@@ -130,8 +130,8 @@ def take_g3_data(S, dur, **stream_kw):
 def stream_g3_on(S, make_freq_mask=False, emulator=False, tag=None,
                  channel_mask=None, filter_wait_time=2, make_datfile=False,
                  downsample_factor=None, downsample_mode=None,
-                 filter_disable=False, stream_type=None, subtype='stream',
-                 enable_compression=None):
+                 filter_disable=False, filter_params=None, stream_type=None,
```

Comment:
I think we should specify `filter_order` and `filter_cutoff` separately instead of passing in a dict. I think usually we'll want to just change the filter_cutoff while using the default filter_order, and how its currently written means you always need to pass both together.